contracts/paymaster/lib/IPaymasterErrors.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.10;
+
+/**
+ * @notice Errors that can occur within the paymaster.
+ */
+interface IPaymasterErrors {
+  error ForwarderNotTrusted(address);
+  error InterfaceNotSupported(string);
+  error InvalidPaymasterDataLength();
+  error InvalidApprovalDataLength();
+  error RelayHubAddressNotSet();
+  error RequiresRelayHubCaller();
+  error ValueTransferNotSupported();
+}
```

Comment:
I put these here vs in `ICustomErrors` as they're only ever used as part of the Paymaster.